EDI/UserInterface/01a_Exports.md

```diff
@@ -0,0 +1,340 @@
+
+
+## Create exporter definition
+
+*Exports > Tab EXPORTS > Button Add*
+
+![Create definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionCreate.png "[Create definition]")
+
+**Create definition**
+
+The *Create definition* view includes the *Settings* tab which contains the fields and functions needed to define a basic definition, see [Settings](#settings-tab).
+
+- *Label*   
+   Enter the name of the exporter definition.
+
+- ![Active](../../Assets/Icons/Toggle.png "[Active]") (Active)   
+   Enable this toggle to activate the exporter definition after you have finished configuring it. Disable this toggle if you no longer want to use the exporter definition.
+
+- *Template engine*   
+   Use this drop-down list to select the engine with which *Actindo* shall process the message. For EDIFACT messages, the **XmlExportTemplateEngine** is used. This field is read-only after you have saved a newly created definition.
+
+- *Default connection*    
+  Edit the connection that you have created for the receiver of the message, see [Create connection](../Integration/01_ManageConnections.md#create-connection).
+  
+- *Access key*  
+   Use the access key to give your business partner a key so that s/he can access your *Actindo* tenant without having to log in to request specific data. You are free to specify any string.
+  
+- [CANCEL]   
+   Click this button to finish the exporter definition creation without saving it. The *Create definition* view is closed.
+
+- [SAVE]   
+   Click this button to save the exporter definition. The *Create definition* view is closed.
+  
+## Edit exporter definition
+
+*DataHub Exporter > Exports > Select export definition*
+
+![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+**Edit definition "definition name"**
+
+The *Edit definition* view contains several tabs that you use to configure the definition. Some tabs are not displayed until you have saved some configuration. In general, the following tabs are available:
+
+- [Settings tab](#settings-tab)
+- [File settings tab](#file-settings-tab)
+- [Data sources tab](#data-sources-tab)
+- [Mapping settings tab](#mapping-settings-tab)
+
+## Settings tab   
+
+*DataHub Exporter > Exports > Select export definition > Tab Settings*
+
+![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+The *Settings* tab is used to edit the basic settings of the exporter definition.
+
+- [CANCEL]   
+   Click this button to finish the exporter definition creation without saving it. The *Create definition* view is closed.
+
+- [SAVE]   
+   Click this button to save the exporter definition. The *Create definition* view is closed.
+
+For all other fields and functions, see [Create exporter definition](#create-exporter-definition).
```

Comment:
Vielleicht Verweis oben, nach dem ersten Satz? For a detailed/complete description of all fields and functions, see... 

Vorsicht, Text anpassen: 
- [CANCEL]   
   Click this button to cancel editing the exporter definition without saving it. The *Edit definition* view is closed.

- [SAVE]   
   Click this button to save the changes to the exporter definition. The *Edit definition* view is closed.

Reply:
done